client/webpack.config.js

```diff
@@ -106,7 +106,8 @@ module.exports = {
       'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
       'process.env.SENTRY_DSN': JSON.stringify(SENTRY_DSN),
       'process.env.ET_ENDPOINT': JSON.stringify(ET_ENDPOINT),
-      'process.env.UPDATES_SERVER_PRODUCT_NAME': JSON.stringify(UPDATES_SERVER_PRODUCT_NAME)
+      'process.env.UPDATES_SERVER_PRODUCT_NAME': JSON.stringify(UPDATES_SERVER_PRODUCT_NAME),
+      'process.env.VERSION': JSON.stringify(pkg.version)
```

Comment:
Should we [get rid of passing the version around dynamically](https://github.com/camunda/camunda-modeler/blob/develop/app/lib/index.js#L82) entirely?

Feels like a code smell to me to access a thing (the version) in two different ways.

Reply:
I think I will just use [Metadata](https://github.com/camunda/camunda-modeler/blob/develop/client/src/util/Metadata.js#L25) instead. Let me adjust the PR.